### ISSUE TITLE
Make date filtering visible in Nebraska

### ISSUE THREAD

**Issue by csprayregen** [pieforproviders/pieforproviders#1090]
## 📋 Summary
After we transition from the low code to the fully automated prototype, enable date filtering features on the NE dashboard to match what is already implemented in IL. 

## 💻 Implementation Details

Before releasing this ticket to users, we should also complete #1720, to minimize their confusion.

Additional enhancement to clarify the user experience are also ticketed but can come after we release date filtering to users. Those are covered in: #1730, #1729, and #1728

> **NOTE TO ENGINEERS**
> 
> Please ensure that you are running Google Lighthouse and/or WebAIM (Wave) on any UI elements in your pull request to check for accessibility issues.
> 
> If it is out of scope to address those accessibility issues in your pull request (because the issue is in a parent component and needs refactoring to work, for example), **you are responsible for noting those issues in your pull request and writing bug tickets to address the issues - make sure you link the new tickets in your pull request**

**Comment by katelovescode** [comment]
@csprayregen and @rebeccakarasiktw is this still blocked by design/needs more info?

**Comment by csprayregen** [comment]
@katelovescode I don't think it's blocked. There are some open questions above, but I'm hoping we can release this and and then address those in follow up tickets. Would love for you two to review those questions and make sure you agree

**Comment by katelovescode** [comment]
>  When I look at previous months, the hours used per week column will not appear currently. We believe it will show the current date, regardless of date filtering settings, which is confusing for the user. OPEN QUESTION: should the column disappear as a whole? Just be blank?

As far as I remember, the decision we made was that it will always show the *last week of the month* in prior months without any way to change it.

> If I look at a previous month for which there is no family fee information, what does the user see? Assume family fee is $0 for that month? Say unknown or n/a?

I think it defaults to $0

**Comment by csprayregen** [comment]
@katelovescode please go through the open questions above and document what the default or current state is, i.e. if we turn on date filtering, what will the user see? We can then determine next steps

**Comment by katelovescode** [comment]
Answers to the above cc @csprayregen and @rebeccakarasiktw 

>  When I look at previous months, the hours used per week column will not appear currently. We believe it will show the current date, regardless of date filtering settings, which is confusing for the user. OPEN QUESTION: should the column disappear as a whole? Just be blank?

Currently this does show the current week, the above "we believe" statement is correct.

>  If I look at a previous month for which there is no family fee information, what does the user see? Assume family fee is $0 for that month? Say unknown or n/a?

It will return $0 if there is no family fee (i.e. if there's no active approval for the date, or if the family fee wasn't entered, it will be $0)
